tests/unit/util/test_getter.py

```diff
@@ -108,6 +108,14 @@ def test_getter_expands_only_uppercase_variable_names_with_braced_variables(self
         my_getter = GetterFactory.from_string(str(testfile))
         assert my_getter.get() == "this is my mytoken, and this is my ${not_a_token}"
 
+    def test_getter_ignores_list_comparison_logprep_list_variable(self, tmp_path):
+        os.environ.update({"PYTEST_TEST_TOKEN": "mytoken"})
```

Comment:
Shouldn't the environment have the variable `LOGPREP_LIST` for this test? After all, you want to test here that the blocklist is working, don't you?

Reply:
yes... that is what I test. And for that, the LOGPREP_LIST should not be defined in the actual environment. so that it persists in the output to ensure this is handled by the list_comparison

Reply:
ah I still don't fully understand this. But what if a user tries to set the env variable `LOGPREP_LIST` for some reason?

Reply:
then it is ignored as stated in the documentation

Reply:
but it's not :D If I add the `LOGPREP_LIST` variable into the test like:
```
os.environ.update({"PYTEST_TEST_TOKEN": "mytoken", "LOGPREP_LIST": "foo"})
```
Then test fails and I get the following error: 
```
AssertionError: assert 'this is my mytoken, and this is my foo' == 'this is my mytoken, and this is my ${LOGPREP_LIST}'
```

Reply:
you are right... I have to fix it

Reply:
done